.github/workflows/commit-message.yml

```diff
@@ -0,0 +1,13 @@
+name: Commit Message
+on: [pull_request]
+jobs:
+  golangci:
+    name: commit-message
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Check if JIRA ID is in the commit message
```

Comment:
@slaskawi could you please explain this step? When the Jira ID is missing the build will fail?

Reply:
Yes, I believe @pb82 asked about exactly the same here (but using different words): https://github.com/keycloak/keycloak-operator/pull/213#discussion_r445503338. Let's continue discussion there.